### PR TITLE
fix: [#176097298] card showing the bottom borders

### PR DESCRIPTION
**Pull request by qesya**
## Short description
implements on the credit cards, the same style as for the bancomat card
Screenshot 2021-01-20 at 11 03 36

### CODE REVIEW

**Comment by thisisjp** [comment]
@qesya @Undermaken the card should have the same aspect ratio of the PagoBANCOMAT card; I guess the shorter height here is due to the lack of the Bank logo. Is there a way to have all the cards (credit, bancomat, other methods) looking the same? Thanks :)

**Review by Undermaken — CHANGES_REQUESTED** [review]
https://github.com/pagopa/io-app/pull/2653#issuecomment-752006230
I guess it should be fixes

**Comment by qesya** [comment]
> @qesya @Undermaken the card should have the same aspect ratio of the PagoBANCOMAT card; I guess the shorter height here is due to the lack of the Bank logo. Is there a way to have all the cards (credit, bancomat, other methods) looking the same? Thanks :)

just updated, thanks for the tip

File: ts/screens/wallet/TransactionsScreen.tsx [hunk 1]
```diff
@@ -106,19 +125,21 @@ class TransactionsScreen extends React.Component<Props> {
...
-          hideMenu={false}
-          isFavorite={isFavorite}
-          onSetFavorite={(willBeFavorite: boolean) =>
-            handleSetFavourite(willBeFavorite, () =>
-              this.props.setFavoriteWallet(selectedWallet.idWallet)
-            )
-          }
-          onDelete={() => this.props.deleteWallet(selectedWallet.idWallet)}
-        />
+        <View style={styles.cardBox}>
```
Comment (debiff) [inline ts/screens/wallet/TransactionsScreen.tsx:128]: There is a problem on android

You can try to use the same solution of: https://github.com/pagopa/io-app/blob/4e4e1b6640f0d8f996debf3411b86f86456f1030/ts/features/wallet/component/BaseCardComponent.tsx#L63

**Review by debiff — CHANGES_REQUESTED** [review]

File: ts/components/wallet/card/CardComponent.style.ts [hunk 2]
```diff
@@ -3,14 +3,26 @@ import { makeFontStyleObject } from "../../../theme/fonts";
...
     shadowOffset: { width: 0, height: 2 },
     shadowOpacity: 0.1,
     shadowRadius: 1.5,
-    elevation: -7,
-    zIndex: -7,
+    elevation: Platform.OS === "android" ? -2 : 25,
+    zIndex: Platform.OS === "android" ? -2 : 25
+  },
+  card: {
+    // iOS and Andorid card shadow
```
Comment (debiff) [inline ts/components/wallet/card/CardComponent.style.ts:25]: This comment seems wrong

File: ts/components/wallet/card/CardComponent.style.ts [hunk 3]
```diff
@@ -3,14 +3,26 @@ import { makeFontStyleObject } from "../../../theme/fonts";
 import variables from "../../../theme/variables";
 
 export default StyleSheet.create({
-  card: {
-    // iOS and Andorid card shadow
+  cardHeader:{
+    backgroundColor: variables.brandGray,
```
Comment (debiff) [inline ts/components/wallet/card/CardComponent.style.ts:7]: Repeated prop. Already present in `card` class
```suggestion
```

File: ts/components/wallet/card/CardComponent.style.ts [hunk 4]
```diff
@@ -3,14 +3,26 @@ import { makeFontStyleObject } from "../../../theme/fonts";
...
+    marginTop: -20,
+    borderRadius: 8,
```
Comment (debiff) [inline ts/components/wallet/card/CardComponent.style.ts:9]: Same as `backgroundColor`

File: ts/components/wallet/card/CardComponent.style.ts [hunk 5]
```diff
@@ -3,14 +3,26 @@ import { makeFontStyleObject } from "../../../theme/fonts";
...
+    marginLeft: 12,
+    marginRight: 12,
+    paddingBottom: 10,
+    borderBottomLeftRadius: 8,
```
Comment (debiff) [inline ts/components/wallet/card/CardComponent.style.ts:13]: Why do you use both `borderBottomLeftRadius`/`borderBottomRightRadius` and `borderRadius`?

**Review by debiff — CHANGES_REQUESTED** [review]

**Review by debiff — APPROVED** [review]

**Pull request merged**